src/System.Private.ServiceModel/tools/PRService/web.config

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
```

Comment:
Sounds good. We can also use it to bump up ASP.NET default request timeout (executionTimeout) if we see timeout becomes an issue in production.